package-lock.json

```diff
@@ -1,10 +1,11 @@
 {
-  "name": "ansible-language-server",
+  "name": "@ansible/ansible-language-server",
```

Comment:
This is addressed in #11, it should be separate.

Reply:
I took it out, is now part of #25